File: sdks/java/basys.sdk/src/main/java/org/eclipse/basyx/submodel/metamodel/connected/submodelelement/ConnectedDataElement.java
package org.eclipse.basyx.submodel.metamodel.connected.submodelelement;

import org.eclipse.basyx.submodel.metamodel.api.submodelelement.IDataElement;
import org.eclipse.basyx.vab.modelprovider.VABElementProxy;

/**
 * "Connected" implementation of DataElement
 * 
 * @author rajashek
 *
 */
public class ConnectedDataElement extends ConnectedSubmodelElement implements IDataElement {
	public ConnectedDataElement(VABElementProxy proxy) {
		super(proxy);
	}
}

File: sdks/c++/basys.sdk.cc/src/submodel/submodel/connected/submodelelement/ConnectedDataElement.cpp
/*
 * ConnectedDataElement.cpp
 *
 *      Author: wendel
 */

#include <BaSyx/submodel/connected/submodelelement/ConnectedDataElement.h>
#include <BaSyx/shared/types.h>

#include <memory>


namespace basyx {
namespace submodel {

ConnectedDataElement::ConnectedDataElement(std::shared_ptr<vab::core::proxy::IVABElementProxy> proxy) :
  ConnectedSubmodelElement(proxy)
{}

}
}
